Expand on applicability of this scheme for clarity (#1806)
httpwg/http-extensions: draft-ietf-httpbis-priority.md

# Applicability of the Extensible Priority Scheme

The priority scheme defined by this document is primarily focused on the
prioritization of HTTP response messages (see {{Section 3.4 of HTTP}}). It

> defines new priority parameters ({{parameters}}) and signals ({{header-field}}

Author: WFM

and {{frame}}) intended to communicate the priority of responses to a server
that is responsible for prioritizing them. {{server-scheduling}} provides
considerations for servers about acting on those signals in combination with
other inputs and factors.

The CONNECT method (see {{Section 9.3.6 of HTTP}}) can be used to establish
tunnels. Signaling applies similarly to tunnels; additional considerations for
server prioritization are given in {{connect-scheduling}}.

{{client-scheduling}} describes how clients can optionally apply elements of
this scheme locally to the request messages that they generate.

> Some forms of HTTP extensions change HTTP/2 or HTTP/3 stream behavior or
> define new data carriage mechanisms. Such extensions are not considered by this
> document; they can define themselves how this priority scheme might be applied.

Reviewer: ```suggestion
Some forms of HTTP extensions might change HTTP/2 or HTTP/3 stream behavior or
define new data carriage mechanisms. Such extensions can define themselves how
this priority scheme is to be applied.
```

I would assume that this is hand-waving to extensions that are _yet_ to be standardized. Then, we "cannot" consider how this prioritization scheme will be applied to them.

Author: much neater, thanks!
